tests/basic/storage_pool_test.go

```diff
@@ -4172,6 +4343,39 @@ var _ = Describe("{AddNewPoolWhileFullPoolExpanding}", func() {
 
 })
 
+func adjustReplPools(firstNode, replNode node.Node, isjournal bool) error {
+
+	selectedNodeSize := getTotalPoolSize(firstNode)
+	secondReplSize := getTotalPoolSize(replNode)
+	if secondReplSize <= selectedNodeSize {
+		secondPool := replNode.StoragePools[0]
+		maxSize := secondPool.TotalSize / units.GiB
+		for _, p := range replNode.StoragePools {
+			currSize := p.TotalSize / units.GiB
+			if currSize > maxSize {
+				maxSize = currSize
+				secondPool = p
+			}
+		}
+
+		expandSize := maxSize * 2
+		log.InfoD("Current Size of the pool %s is %d", secondPool.Uuid, secondPool.TotalSize/units.GiB)
+		err := Inst().V.ExpandPool(secondPool.Uuid, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, expandSize)
+		if err != nil {
+			err = fmt.Errorf("pool expansion init failed for %s. Err : %v", secondPool.Uuid, err)
+			return err
```

Comment:
Can we just do this, don't need to define `err` and then return it
```
		if err := Inst().V.ExpandPool(secondPool.Uuid, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, expandSize); err != nil {
                           return fmt.Errorf("pool expansion init failed for %s. Err : %v", secondPool.Uuid, err)
                }
```

Reply:
fixed